docs/src/main/sphinx/release/release-391.md

```diff
@@ -0,0 +1,16 @@
+# Release 391 (20 July 2022)
+
+## General
+
+* Improve performance of repeated aggregations with ``CASE`` expressions. ({issue}`12548`)
+
+## BigQuery connector
+
+* Add support for reading external tables. ({issue}`13164`)
+* Add support for specifying a table comment and column comments when creating a
+  table. ({issue}`13105`)
+
+## Hive connector
+
+* Add support for
+  [AWS Athena partition projection](https://docs.aws.amazon.com/athena/latest/ug/partition-projection.html). ({issue}`11305`)
```

Comment:
Lets not link there... instead we have to make sure the documentation PR is merged and then we link there.